staging/src/k8s.io/sample-apiserver/docs/minikube-walkthrough.md

```diff
@@ -1,56 +1,67 @@
 # Minikube walkthrough
 
 This document will take you through setting up and trying the sample apiserver on a local minikube from a fresh clone of this repo.
+You can optionally run and develop the sample apiserver on your local machine and use minikube to host the etcd.
```

Comment:
and use minikube only to host the etcd.

Reply:
But why don't you run etcd locally as well? Isn't that much easier?

Reply:
It's another binary you need to install and put in your `PATH`. I find it easier to run in Docker or Kubernetes.